index.d.ts

```diff
@@ -0,0 +1,27 @@
+declare module "cz-customizable" {
+  export interface Option {
+    name: string;
+    value?: string;
+  }
+  export interface Options {
+    types: Option[];
+    scopes?: Option[];
```

Comment:
Scopes is an array of `{name: 'accounts'},`. Is this something you could add to the types? 
Here is a file example: https://github.com/leonardoanalista/cz-customizable/blob/master/cz-config-EXAMPLE.js
Note: I am not very familiar with Typescript. Please let me know if is not possible ok?

Reply:
I guess the same for types, and array of `{value: 'feat',     name: 'feat:     A new feature'}`

Reply:
possibly `allowBreakingChanges` too?

Reply:
I've defined `types` and `scopes` as type `Option[]`. `Option` is defined as `{ value?: string, name: string }` where `value` is optional (the "?" means the property is optional). I believe this matches your examples. Is this typing incorrect?

In my use case I actually defined both a `name` and `value` for `scopes` so I could have explanatory text just like we have for `types`.

`allowBreakingChanges` is an array of strings in your example. Can it be something else? The one bit that is missing is that I don't think I can enforce is that these strings have to match the  type names.

Reply:
yes, it is a type of `Option`. I missed that. Cool thanks.